.gitignore

```diff
@@ -206,7 +206,6 @@ tools/escape_mapper/escape_mapper
 BUILDS
 DEBUG
 RELEASE
-cmake-build-*
```

Comment:
Can we keep this? CLion uses this prefix by default

Reply:
Sure.